src/AWSMetadata.jl

```diff
@@ -100,8 +100,8 @@ function _generate_high_level_wrapper(services::AbstractArray{<:OrderedDict})
             println(f, "# This file is auto-generated by AWSMetadata.jl")
             println(f, "using AWS")
             println(f, "using AWS.AWSServices: $service_name\n")
```

Comment:
The separation between the imports isn't necessary anymore
```suggestion
            println(f, "using AWS.AWSServices: $service_name")
```